src/preferences.py

```diff
@@ -208,8 +208,10 @@ def apply(self):
             self.addequation_x_stop.get_text()
         preferences["addequation_step_size"] = \
             self.addequation_step_size.get_text()
-        preferences["clipboard_length"] = self.clipboard_length.get_value()
-        preferences["export_figure_dpi"] = self.export_figure_dpi.get_value()
+        preferences["clipboard_length"] = \
+            int(self.clipboard_length.get_value())
```

Comment:
There's some overlap with the PR I pushed, but it's also good to set these types here to keep compatibility with older config files (for people upgrading)